src/main/java/net/rebworks/lunchy/domain/parsers/Hildas.java

```diff
@@ -26,20 +30,32 @@ public Hildas(final DateCalculator dateCalculator) {
     List<Lunch> lunches = new ArrayList<>();
 
 
-    JSONObject jsonObject = new JSONObject(input);
+    ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
It would have been better to use an injected object mapper here (I'm 99% sure one is accessible in the injection context). Also it would be better to have it as a field instead of initializing in every call.

However I will turn a blind eye to that for now :nerd_face: